src/main/java/frc/robot/RobotContainer.java

```diff
@@ -90,7 +76,87 @@
   // Initializes the default commands and command bindings.
   public RobotContainer() {
     // Configure the automous chooser.
-    m_autoChooser.setDefaultOption("Example Autonomous Trajectory", m_exampleAutoCommand);
+
+    // Add the reset odometry command here, in case the Shuffleboard button breaks (hint: in
+    // simulation, after it fails to quit peacefully, it will break).
+    m_autoChooser.addOption("Reset Odometry", m_resetOdometryCommand);
+
+    // DriveCommand tests:
```

Comment:
This entire PR looks good to me, unless you find it appropriate to remove these test commands commented out.

Reply:
They're fine to stay.